db/main.py

```diff
@@ -0,0 +1,68 @@
+from fastapi import FastAPI, Depends, HTTPException
+from sqlalchemy.orm import Session
+
+import crud
+import schemas
+from db.database import SessionLocal
+
+app = FastAPI()
+
+
+def get_db() -> Session:
+    db = SessionLocal()
+
+    try:
+        yield db
+    finally:
+        db.close()
+
+
+@app.get("/")
+def root() -> dict:
+    return {"message": "Hello form Library"}
```

Comment:
There's a typo in the root endpoint message. It should be 'Hello from Library' instead of 'Hello form Library'.